script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -326,7 +324,7 @@ I thought it was one of those articulated wooden dolls with strings attached;
 [sha:4a84c0d437fd80a5ae1a30b0da010258ea4d225d]{
 -- Page 646, Offset 25996. Glued. Mods: @x.
 -- おい、下がれノエル！　首ねじ切られるぞ！」
-Oi, get back Noel! You'll break your damn neck!"
+'Ey! Stand down, Noel! Your gonna lose your fuckin' head!"
```

Comment:
I'd probably prefer oi to ey unless Mario is from New York

Reply:
Fairly certain "oi" isn't American English.

Reply:
you're, unless its for intended effect.